transport-sctp/src/main/java/io/netty/channel/sctp/oio/OioSctpChannel.java

```diff
@@ -184,8 +185,8 @@ protected int doReadMessages(List<Object> msgs) throws Exception {
 
         Set<SelectionKey> reableKeys = readSelector.selectedKeys();
         try {
-            for (SelectionKey ignored : reableKeys) {
-                RecvByteBufAllocator.Handle allocHandle = unsafe().recvBufAllocHandle();
+            for (@SuppressWarnings("unused") SelectionKey ignored : reableKeys) {
```

Comment:
Not sure why this loop is necessary...see https://github.com/netty/netty/issues/3884